packages/fxa-shared/subscriptions/subscriptions.ts

```diff
@@ -3,14 +3,17 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
This is definitely one of my least favorite names for a file.... not sure what I'd expect to be in here. The contents here are all Type Guards however, so maybe we just rename this to `type-guards.ts` and call it a day? 😁

Reply:
Definitely didn't touch this, but since you included an emoticon at the end of your request, I'll go ahead and rename the file.